Add table of JVM to Unicode (#944)
neo4j/docs-cypher: modules/ROOT/pages/syntax/parsing.adoc

M\u0041TCH (m) RETURN m;
----

> The Unicode Version used by Cypher is coupled with the JVM version that is running.

Reviewer: ```suggestion
The Unicode version used by Cypher depends on the running JVM version.
```

[options="header", cols="1,2,3"]
|===

> | Neo4j Version	| JVM compliancy                                   | Unicode Version

Reviewer: ```suggestion
| Neo4j version	| JVM compliancy                                   | Unicode version
```
